### PR TITLE
Template fixes

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

**Comment by ostcar** [comment]
ok

File: openslides/agenda/static/templates/agenda/item-list.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h1 translate>Agenda</h1>
               class="btn btn-default form-control"
               ng-click="autoNumbering()">
             <i class="fa fa-sort-numeric-asc"></i>
-            <translate>Auto numbering</translate>
+            <translate>Number</translate>
```
Comment (normanjaeckel) [inline openslides/agenda/static/templates/agenda/item-list.html]: Hm. Is it easy to understand that the whole agenda will be numberd if one clicks here? Proposal `Number agenda`.

Comment (emanuelschuetze) [inline openslides/agenda/static/templates/agenda/item-list.html]: Changed...

**Comment by emanuelschuetze** [comment]
All done. Rebased. Pot file updated. Ready for merge.

**Pull request merged**